README.md

```diff
@@ -1,2 +1,15 @@
 # hello-microverse
-A "Hello world" project
+
+> A "Hello world" project
+
+## Built With
+
+- HTML, CSS
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@amejid](https://github.com/amejid)
+- Twitter: [@amejidoumer](https://twitter.com/amejidoumer)
+- LinkedIn: [amejid](https://linkedin.com/in/amejid)
```

Comment:
- [ ] It would be nice if you make your README file more descriptive. You can use the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse which contains all main headers. Kindly, change it.

Reply:
I have made the necessary changes. The pull request is automatically closed because I renamed the branch.

Reply:
Thank you for reviewing my project, I will make the necessary corrections as suggested.